src/app/shared/components/paginator/paginator.component.ts

```diff
@@ -57,7 +59,7 @@ export class PaginatorComponent implements OnInit, OnChanges {
   }
 
   private getTotalPageAmount(): number {
-    return Math.ceil(this.totalEntities / Constants.ITEMS_PER_PAGE);
+    return Math.ceil(this.totalEntities / Constants.ITEMS_PER_PAGE) || Math.ceil(this.totalEntitiesDir / Constants.ITEMS_PER_PAGE_DIR);
```

Comment:
just as an idea, maybe for getTotalPageAmount method will be good to receive from parent component flag like isDirections and then check and if yes use Constants.ITEMS_PER_PAGE_DIR